arbitrum-docs/faqs/protocol-faqs.mdx

```diff
@@ -0,0 +1,49 @@
+
+If anywhere between 2 and 18 of the committee members are well behaved, the AnyTrust chain operates in "Rollup mode"; i.e., data gets posted on L1.
+
+In what should be the common and happy case, however, in which at least 19 of the 20 committee members are well behaved, the system operates without the posting the L2 chain's data on L1, and thus, users pay significantly lower fees. This is the core upside of AnyTrust chains over rollups.
+
+Variants of the AnyTrust model in which the new trust assumption is minimized are under consideration; stay tuned.
+
+For more, see [Inside AnyTrust](./inside-anytrust.mdx).
+
+### Are “Sequencers” the same thing as “Validators”? Can a centralized Sequencer do bad things like steal all my money? {#q-seq-vs-val}
+
+No and no!
+
+An Arbitrum Chain's Sequencer(s) and Validators and completely distinct entities, with their own distinct roles.
+
+The [Sequencer](./sequencer.mdx) is the entity granted specific privileges over ordering transactions; once the Sequencer commits to an ordering (by posting a batch on Ethereum), it has no say over what happens next (i.e., execution). A malicious/faulty Sequencer can do things like reorder transactions or *temporarily* delay a transaction's inclusion — things which could be, to be sure, annoying and bad — but can do nothing to compromise the chain's safety.
+
+The *Validators* are the ones responsible for the safety of the chain; i.e., making staked claims about the chain state, disputing each other, etc.
+
+Currently, on Arbitrum One, the Sequencer is a centralized entity maintained by Offchain Labs. Eventually, the single Sequencer will be replaced by a distributed committee of Sequencers who come to consensus on transaction ordering. This upgrade be an improvement; we don't want you to have to trust us not to reorder your transactions. However, it also isn't *strictly* necessary for Arbitrum One to achieve its most fundamental properties.
+
+In other words: **_An Arbitrum Rollup chain with a centralized Sequencer could theoretically still be trustless!_**
+
+Which is to say — the more important thing than decentralizing the Sequencer i.e., the thing you ought to care more about — is decentralizing the *Validators*.
+
+Arbitrum One's Validator set is currently whitelisted; overtime, the whitelist will grow and then be removed entirely. For more info see ["Mainnet Beta"](/mainnet-beta.mdx).
+
+### If there is a dispute, can my L2 transaction get reorged / throw out / "yeeted"? {#q-dispute-reorg}
+
+Nope; once an Arbitrum transaction is included on L1, there is no way it can be reorged (unless the L1 itself reorgs, of course). A "dispute" involves Validators disagreeing over execution, i.e., the outputted state of a chain. The inputs, however, can't be disputed; they are determined by the Inbox on L1. (See [Transaction Lifecycle](https://developer.arbitrum.io/tx-lifecycle))
```

Comment:
```suggestion
Nope; once an Arbitrum transaction is included on L1, there is no way it can be reorged (unless the L1 itself reorgs, of course). A "dispute" involves Validators disagreeing over execution, i.e., the outputted state of a chain. The inputs, however, can't be disputed; they are determined by the Inbox on L1. (See [Transaction Lifecycle](/tx-lifecycle.mdx))
```